etl/steps/data/grapher/migration/2024-11-18/migration_between_regions.py

```diff
@@ -0,0 +1,40 @@
+"""Load a garden dataset and create a grapher dataset."""
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load garden dataset.
+    ds_garden = paths.load_dataset("migration_between_regions")
+
+    # Read table from garden dataset
+    tb = ds_garden["migration_between_regions"].reset_index()
+
+    tb = tb.pivot(columns="country_destination", index=["country_origin", "year"], values="migrants_all_sexes")
```

Comment:
Why not doing this in garden?

Reply:
I feel the ["country_destination", "country_origin", "year"] index is more intuitive to work with and we only need the ["country"/"country_origin", "year"] index to make this data set work in grapher. 

Theoretically we could then also index on country destination in this grapher step and get the indicators for migration FROM every country - but we don't need this for now.